server/startup/serverRunning.js

```diff
@@ -13,6 +13,12 @@ Meteor.startup(function() {
 		}
 	}
 
+	var mongoOptionStr = process.env.MONGO_OPTIONS;
+	if (typeof(mongoOptionStr) !== 'undefined') {
```

Comment:
Let's go with `typeof mongoOptionStr` to stay consistent with the rest of our codebase.